src/tracking.rs

```diff
@@ -144,13 +143,7 @@ pub(crate) async fn download_gossip(persistence_sender: mpsc::Sender<DetectedGos
 
 			if needs_to_notify_persister {
 				needs_to_notify_persister = false;
-				let sender = local_persistence_sender.clone();
-				tokio::spawn(async move {
-					let _ = sender.send(DetectedGossipMessage {
-						timestamp_seen: current_timestamp as u32,
-						message: GossipMessage::InitialSyncComplete,
-					}).await;
-				});
+				let _ = local_persistence_sender.send(GossipMessage::InitialSyncComplete).await;
```

Comment:
I didn't find one, but is there a better way not to have an unused result here than just a `let _`? Not sure why there isn't a simple `unwrap()`